content/source/docs/cloud/index.html.md

```diff
@@ -15,11 +15,11 @@ For expediency, this documentation refers to the application as Terraform Cloud.
 
 ## About Terraform Cloud and Terraform Enterprise
 
-[Terraform Cloud](https://www.hashicorp.com/products/terraform/) is an application that helps teams use Terraform together. It manages Terraform runs in a consistent and reliable environment, and includes easy access to shared state and secret data, access controls for approving changes to infrastructure, a private registry for sharing Terraform modules, detailed policy controls for governing the contents of Terraform configurations, and more.
+[Terraform Cloud](https://terraform.io/cloud) is an application that helps teams use Terraform together. It manages Terraform runs in a consistent and reliable environment, and includes easy access to shared state and secret data, access controls for approving changes to infrastructure, a private registry for sharing Terraform modules, detailed policy controls for governing the contents of Terraform configurations, and more.
 
-Terraform Cloud is available as a hosted service at [https://app.terraform.io](https://app.terraform.io). We offer free accounts for small teams, and paid plans with additional feature sets for medium-sized businesses.
+Terraform Cloud is available as a hosted service at [https://app.terraform.io](https://app.terraform.io). Small teams can sign up for free. As these teams scale, they can gain access to collaborative features through paid tiers. Finally, multiple teams within an organization can leverage [the Business tier](https://www.hashicorp.com/products/terraform/editions/cloud), which includes increased security, scalability, and visibility. 
 
-Large enterprises can purchase [Terraform Enterprise](/docs/enterprise/index.html), our self-hosted distribution of Terraform Cloud. It offers enterprises a private instance of the Terraform Cloud application, with no resource limits and with additional enterprise-grade architectural features like audit logging and SAML single sign-on.
+Enterprises with advanced security and compliance needs can purchase [Terraform Enterprise](/docs/enterprise/index.html), our self-hosted distribution of Terraform Cloud. It offers enterprises a private instance that includes the advanced features available in Terraform Cloud.
```

Comment:
should we mention audit logging and SAML SSO? these seem like important features for enterprises